test/parallel/test-eventemitter-asyncresource.js

```diff
@@ -143,17 +143,9 @@ throws(
   { code: 'ERR_INVALID_THIS' }
 );
 
-throws(
-  () => Reflect.get(EventEmitterAsyncResource.prototype, 'asyncId', {}),
-  { code: 'ERR_INVALID_THIS' }
-);
-
-throws(
-  () => Reflect.get(EventEmitterAsyncResource.prototype, 'triggerAsyncId', {}),
-  { code: 'ERR_INVALID_THIS' }
-);
-
-throws(
-  () => Reflect.get(EventEmitterAsyncResource.prototype, 'asyncResource', {}),
-  { code: 'ERR_INVALID_THIS' }
-);
+['asyncId', 'triggerAsyncId', 'asyncResource'].forEach((member) => {
+  throws(
+    () => Reflect.get(EventEmitterAsyncResource.prototype, member, {}),
+    { code: 'ERR_INVALID_THIS' }
+  );
```

Comment:
Could we include the actual method called into the error message in case the test fails? Line number is no longer useful.

Reply:
Just included a new regex for the stack trace. that would be enough, in case of failure the stack trace assertion will be the guide.

Reply:
Revision: https://github.com/nodejs/node/commit/c27bb19efd8719417b3b87d4e014678bc20062a8